subprojects/docs/src/snippets/dependencyManagement/catalogs-toml/tests/sanityCheck.sample.conf

```diff
@@ -0,0 +1,2 @@
+executable: gradle
```

Comment:
I'm confused why we're adding this file at all now? I would either expect this file not to exist at all, or the other 3 copies to be added (without the disable warning flag)

Reply:
No longer needed, removed.